XamarinCommunityToolkit/Converters/TextCaseConverter.shared.cs

```diff
@@ -7,8 +7,14 @@ namespace XamarinCommunityToolkit.Converters
     /// <summary>
     /// Converts text (string, char) to certain case.
     /// </summary>
-    public class TextCaseConverter : IValueConverter
+    [ContentProperty(nameof(Type))]
+    public class TextCaseConverter : ValueConverterMarkupExtension, IValueConverter
     {
+        /// <summary>
+        /// The desired text case that the text should be converted to.
+        /// </summary>
+        public TextCaseConverterType Type { get; set; }
```

Comment:
Maybe rename this enum to not reference the converter itself (which is a technical thing) but the concept it represents e.g. `TextCaseType`.